<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Client</class>
 <widget class="QWidget" name="Client">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1440</width>
    <height>1024</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1440</width>
    <height>1024</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Client</string>
  </property>
  <widget class="QLineEdit" name="LoginInput">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>500</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>70</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Логин</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PwdInput">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>590</y>
     <width>600</width>
     <height>70</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>104</width>
     <height>70</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>600</width>
     <height>70</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Пароль</string>
   </property>
  </widget>
  <widget class="QPushButton" name="LoginButton">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>730</y>
     <width>231</width>
     <height>81</height>
    </rect>
   </property>
   <property name="text">
    <string>Войти</string>
   </property>
  </widget>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1440</width>
     <height>1024</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../../Resources.qrc">:/images/light/auth_screen.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="themeSwitchButton">
   <property name="geometry">
    <rect>
     <x>1320</x>
     <y>0</y>
     <width>121</width>
     <height>151</height>
    </rect>
   </property>
   <property name="focusPolicy">
    <enum>Qt::FocusPolicy::NoFocus</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>background</zorder>
  <zorder>LoginInput</zorder>
  <zorder>PwdInput</zorder>
  <zorder>LoginButton</zorder>
  <zorder>themeSwitchButton</zorder>
 </widget>
 <resources>
  <include location="../../Resources.qrc"/>
 </resources>
 <connections/>
</ui>
